src/AdminConsole/Components/App.razor

```diff
@@ -7,6 +7,7 @@
     <base href="/"/>
     <SecureStylesheet href="css/site.css" />
     <SecureStylesheet href="css/tailwind.css" />
+    <SecureStylesheet href="Passwordless.AdminConsole.styles.css" />
```

Comment:
This file will contain all the isolated CSS or scoped CSS for the individual components. For example for `Create.razor.css`. This is generated at build time.